tarodb/sqlc/queries/assets.sql

```diff
@@ -459,6 +459,9 @@ WITH target_asset(asset_id) AS (
     JOIN script_keys 
         ON assets.script_key_id = script_keys.script_key_id
     WHERE script_keys.tweaked_script_key = $1
+    -- TODO(guggero): Fix this by disallowing multiple assets with the same
```

Comment:
There can be no duplicate `tweaked_script_key` values in the `script_keys` table, that's correct. But there can be multiple assets that reference the same script key (`script_key_id INTEGER NOT NULL REFERENCES script_keys(script_key_id),`) which is the problem this solves.

Reply:
Hmm, yeh though fwiw we don't really need to update the proofs for the NUMs outputs, since they just sort of hang around on chain. 

What was the failure/issue we ran into w/o the `LIMIT 1` here? Just wanting to understand the interaction (and divergence between `sqlite` and `postgres`) a bit more.

Reply:
The issue was that a sub query returning more than 1 result when inserting failed on Postgres while it didn't on sqlite. I assume sqlite just took the first result but Postgres is more strict.